### ISSUE TITLE
Unable to open in-player chapter controls while using a screen-reader.

### ISSUE THREAD

**Issue by Murcielago7384** [OdyseeTeam/odysee-frontend#3547]
**Bug**
I use NVDA, a free and open source screen-reader for desktop. When I have a video displayed on Odysee (full screen), I am able to find the current chapter button by pressing tab to cycle through the various player controls, such as play, mute, volume, et cetera. Pressing enter on any of these activates the control as you would expect, apart from the 'chapters' button. Previously, this would bring up a small check-box style menu that I could then use my arrow keys to cycle through. However, as it stands now, the only way to cycle through chapters is to go to the list below the description box.
This issue is occurring on the desktop version of the website, and appears to occur in both Firefox and Chromium based browsers.

**To Reproduce**
1. Once you have the screen-reader up and running, go to the desktop site of Odysee on any browser.
2. Go to any video you know to have chapter segments, and play that video in full screen mode.
3. Use tab to cycle through the player controls. Note that, by default, NVDA might be in its focus mode. Switch modes by pressing spacebar in conjunction with your NVDA key (which will either be insert or caps lock), until you hear a low beep.
4. Press enter once NVDA reaches the chapters button. Nothing will happen.

**Expected behavior**
As stated above, previously, a checkbox-style menu would appear, which could then be arrowed through using up and down arrows. However, this button is no longer able to be activated while using a screen-reader.

**System Configuration**
Windows 11. Any browser, though primarily tested on LibreWolf and Helium. This is the version of the desktop website as of 4 August 2026.

**Additional context**
NVDA is available from the NV Access website. This screen-reader is open source and has lots of documentation detailing how to make apps/websites accessible.
